flink-runtime/src/main/java/org/apache/flink/runtime/taskmanager/Task.java

```diff
@@ -134,120 +134,190 @@
  */
 public class Task implements Runnable, TaskSlotPayload, TaskActions, PartitionProducerStateProvider, CheckpointListener, BackPressureSampleableTask {
 
-	/** The class logger. */
+	/**
+	 * The class logger.
+	 */
```

Comment:
All changes in this file seem to be quite unrelated. Please revert.

Reply:
It was accidentally added. But there is actually a change that is relevant:
```
@@ -755,7 +836,7 @@ public class Task implements Runnable, TaskSlotPayload, TaskActions, PartitionPr
                        // an exception was thrown as a side effect of cancelling
                        // ----------------------------------------------------------------

-                       t = TaskManagerExceptionUtils.tryEnrichTaskManagerError(t);
+                       TaskManagerExceptionUtils.tryEnrichTaskManagerError(t);
```
I reverted everything except for the change above.